.github/workflows/androidBump.yml

```diff
@@ -17,8 +17,8 @@ jobs:
         uses: ./.github/actions/composite/setupNode
 
       - name: Setup Ruby
-        # v1.190.0
-        uses: ruby/setup-ruby@a6e6f86333f0a2523ece813039b8b4be04560854
+        # v1.229.0
+        uses: ruby/setup-ruby@354a1ad156761f5ee2b7b13fa8e09943a5e8d252
```

Comment:
So in the future if we need to update these, is there a process documented somewhere?

Reply:
@roryabraham do you know? The process I followed was go to the repo, find the latest version, look at the commit.